docker/Dockerfile

```diff
@@ -101,12 +101,17 @@ RUN git clone -b ${TORCH_VISION_BRANCH} https://github.com/pytorch/vision.git &&
     pip install dist/*.whl && cd ..
 ARG TORCH_BENCH_BRANCH=main
 ARG TORCH_BENCH_COMMIT=default
+ARG TRANSFORMERS_VERSION=main
```

Comment:
let's change the default value to `default`, and add if else block in below, if the value is default, we used the pined commits from pytorch, others we can try to `pip install transformers==${TRANSFORMERS_VERSION}` directly, how do you think?

Reply:
Done